tests/integration/cloud/rest_contracts/unit_test_mocks/conftest.py

```diff
@@ -1,131 +0,0 @@
-import json
```

Comment:
I know this isn't used right now, but can we leave this in? The idea is to use mocks like this in unit tests once we know how request/response should look.